queue.go

```diff
@@ -511,3 +510,17 @@ func (q *Queue) receiveJob(ctx context.Context) *amqp.Delivery {
 		return &job
 	}
 }
+
+func (q *Queue) setUrl() {
+	const urlString = "amqp://%s:%s@%s:%s/%s"
+	hostName := utils.GetEnvVar("RABBITMQ_HOSTNAME", "")
+	port := utils.GetEnvVar("RABBITMQ_PORT", "")
+	userName := utils.GetEnvVar("RABBITMQ_USERNAME", "")
+	password := utils.GetEnvVar("RABBITMQ_PASSWORD", "")
+	vHost := utils.GetEnvVar("RABBITMQ_VHOST", "")
+	if hostName == "" || port == "" || userName == "" || password == "" {
```

Comment:
Why don't you check `vHost`?

Reply:
@krisowski i had put the check 
bit while testing it with worker i found out that the url used bu worker is not using vhost so i removed the check for now

Reply:
Yes, today `worker` assumes that `VHOST` is `/` but this is wrong. This check will be needed anyway so why did you remove it?

Reply:
@krisowski   i have added the required to vhost but the worker is not consuming message